programs/relayer/src/modules/cosmos_to_eth.rs

```diff
@@ -87,7 +87,7 @@ impl CosmosToEthRelayerModuleService {
         let tm_listener = cosmos_sdk::ChainListener::new(tm_client.clone());
 
         let provider = RootProvider::builder()
-            .on_builtin(&config.eth_rpc_url)
+            .connect(&config.eth_rpc_url)
```

Comment:
curious what is the difference

Reply:
fixed

Reply:
I don't think this is changed. But idk if there is anything to fix

Reply:
I must have written this on the wrong comment 🤔